tests/e2e/testfiles/storage_csm_powerflex.yaml

```diff
@@ -11,13 +11,13 @@ spec:
       # Allowed values: ReadWriteOnceWithFSType, File , None
       # Default value: ReadWriteOnceWithFSType
       fSGroupPolicy: "File"
-    configVersion: v2.6.0
+    configVersion: v2.7.0
     replicas: 1
     dnsPolicy: ClusterFirstWithHostNet
     forceUpdate: false
     forceRemoveDriver: true
     common:
-      image: "dellemc/csi-vxflexos:v2.6.0"
+      image: "dellemc/csi-vxflexos:latest"
```

Comment:
shouldn't it be nightly tag instead?

Reply:
Hi @JacobGros We are okay with any tag but since we are merging it into main, I am considering latest one to be more stable wrt nightly.